### PR TITLE
Refactor Header + animation components

### PR DESCRIPTION
**Pull request by mikedemarais**
- refactor `FlyInAnimation` and `OpacityToggle` animations components so they are way simpler and use `react-native-redash`
- Cleanup Fab components & remove dead draggable Fab code
- Creates new `🪐 Discover` header button on WalletScreen 
- deletes fucking toooooons of dead files / dependancies

Fixes RAI-683 https://linear.app/rainbow/issue/RAI-683/add-new-🪐-discover-button-to-walletscreen-header

### CODE REVIEW

**Comment by christianbaroni** [comment]
crashing for me
Screen Shot 2020-06-05 at 7 22 04 PM

**Comment by mikedemarais** [comment]
> crashing for me
> Screen Shot 2020-06-05 at 7 22 04 PM
@christianbaroni 
yeah, i just changed this PR to be a draft for now.. im currently investigating this crash

**Comment by christianbaroni** [comment]
cool, edited my comment, realized it wasn't a full crash

**Comment by christianbaroni** [comment]
also:
Simulator Screen Shot - iPhone 11 Pro - 2020-06-05 at 19 26 57

File: src/components/activity-list/ActivityListHeader.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-    showDivider: false,
-    titleRenderer,
-  }),
-  onlyUpdateForKeys(['title'])
-)(ListHeader);
+const ActivityListHeader = props => (
+  <ListHeader
+    {...props}
+    isSticky
+    showDivider
```
[the remark this answers is not in the document]
Comment (mikedemarais) [reply src/components/activity-list/ActivityListHeader.js:16]: lol oops! thanks!

**Pull request merged**